src/Stryker.Core/Stryker.Core/Mutators/MathMutator.cs

```diff
@@ -0,0 +1,125 @@
+            [MathExpression.Acosh] = new[] { MathExpression.Acos, MathExpression.Asinh, MathExpression.Atanh },
+            [MathExpression.Asin] = new[] { MathExpression.Asinh, MathExpression.Acos, MathExpression.Atan },
+            [MathExpression.Asinh] = new[] { MathExpression.Asin, MathExpression.Acosh, MathExpression.Atanh },
+            [MathExpression.Atan] = new[] { MathExpression.Atanh, MathExpression.Acos, MathExpression.Asin },
+            [MathExpression.Atanh] = new[] { MathExpression.Atan, MathExpression.Acosh, MathExpression.Asinh },
+            [MathExpression.BitDecrement] = new[] { MathExpression.BitIncrement },
+            [MathExpression.BitIncrement] = new[] { MathExpression.BitDecrement },
+            [MathExpression.Ceiling] = new[] { MathExpression.Floor },
+            [MathExpression.Cos] = new[] { MathExpression.Cosh, MathExpression.Sin, MathExpression.Tan },
+            [MathExpression.Cosh] = new[] { MathExpression.Cos, MathExpression.Sinh, MathExpression.Tanh },
+            [MathExpression.Exp] = new[] { MathExpression.Log },
+            [MathExpression.Floor] = new[] { MathExpression.Ceiling },
+            [MathExpression.Log] = new[] { MathExpression.Exp, MathExpression.Pow },
+            [MathExpression.MaxMagnitude] = new[] { MathExpression.MinMagnitude },
+            [MathExpression.MinMagnitude] = new[] { MathExpression.MaxMagnitude },
+            [MathExpression.Pow] = new[] { MathExpression.Log },
+            [MathExpression.ReciprocalEstimate] = new[] { MathExpression.ReciprocalSqrtEstimate },
+            [MathExpression.ReciprocalSqrtEstimate] = new[] { MathExpression.ReciprocalEstimate, MathExpression.Sqrt },
+            [MathExpression.Sin] = new[] { MathExpression.Sinh, MathExpression.Cos, MathExpression.Tan },
+            [MathExpression.Sinh] = new[] { MathExpression.Sin, MathExpression.Cosh, MathExpression.Tanh },
+            [MathExpression.Tan] = new[] { MathExpression.Tanh, MathExpression.Cos, MathExpression.Sin },
+            [MathExpression.Tanh] = new[] { MathExpression.Tan, MathExpression.Cosh, MathExpression.Sinh }
+        };
+
+        /// <summary> Apply mutations to an <see cref="InvocationExpressionSyntax"/> </summary>
+        public override IEnumerable<Mutation> ApplyMutations(InvocationExpressionSyntax node) => node.Expression switch
+        {
+            MemberAccessExpressionSyntax memberAccess => ApplyMutationsToMemberCall(node, memberAccess),
+            IdentifierNameSyntax methodName => ApplyMutationsToDirectCall(node, methodName),
+            _ => Enumerable.Empty<Mutation>()
+        };
+
+        private static IEnumerable<Mutation> ApplyMutationsToMemberCall(InvocationExpressionSyntax node, MemberAccessExpressionSyntax memberAccessExpressionSyntax)
+        {
+            if (memberAccessExpressionSyntax.Expression is not IdentifierNameSyntax memberName ||
+                !memberName.Identifier.ValueText.StartsWith("Math"))
```

Comment:
Maybe at least it's possible to verify the "System" namespace here for it?

Reply:
From my point of view, it looks like we can't check the fully qualified name without the semantic model. But maybe I'm wrong.

Reply:
yes and no; you can check the given name, so you can verify that the name is either `Math.Con` or `Con` (no namespace provided) not 'Something.Con` nor `Something.Math.Con`. Less precise than using the semantic model, but good enough for most cases

Reply:
@dupdob But the existing code already works like that, doesn't it? The code exactly checks what you described.

Reply:
Alrighty then, you can resolve this.